ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -66,6 +67,39 @@ Verify User Can Create, Run and Delete A DS Pipeline From DS Project Details Pag
     Delete Pipeline Server    ${PRJ_TITLE}
     Delete Data Science Project    ${PRJ_TITLE}
 
+Verify No Metadata Pods Were Deployed
+    [Documentation]    Because the product includes code that is currently deactivated, it is imperative to prevent any
```

Comment:
Could you rewrite the text according to the style guide? e.g. _Verifies that metadata pods are not created when running a data science pipeline, as this feature is currently disabled._